src/strategy/mod.rs

```diff
@@ -89,10 +89,16 @@ impl UpdateStrategy {
             UpdateStrategy::FleetLock(_) => self.configuration_label().to_string(),
             UpdateStrategy::Immediate(_) => self.configuration_label().to_string(),
             UpdateStrategy::Periodic(p) => format!(
-                "{}, total schedule length {} minutes (next window {})",
```

Comment:
This `format!` logic became large enough that it can be moved to its own `p.calendar_summary()` or similar.